### DISCUSSION TITLE
BAM file Error on the antismash fungi webserver

### DISCUSSION THREAD

**Discussion by ysahin1** [antismash/antismash#718] [General]
Hello,

I am trying to run antismash fungi on the website, but I got error for my GFF3 file. I would like to know what the wrong about my GFF file is 

Here is an example and  i can also share the GFF file:

##gff-version 3								
Chr01	maker	gene	160	844	.	-	.	ID=EcMO1g00000010;Name=EcMO1g00000010;description=Putative uncharacterized protein
Chr01	maker	mRNA	160	844	.	-	.	ID=EcMO1g00000010-mRNA-1;Parent=EcMO1g00000010;Name=EcMO1g00000010-mRNA-1
Chr01	maker	exon	814	844	.	-	.	ID=EcMO1g00000010-mRNA-1:3;Parent=EcMO1g00000010-mRNA-1
Chr01	maker	exon	509	528	.	-	.	ID=EcMO1g00000010-mRNA-1:2;Parent=EcMO1g00000010-mRNA-1
Chr01	maker	exon	160	414	.	-	.	ID=EcMO1g00000010-mRNA-1:1;Parent=EcMO1g00000010-mRNA-1
Chr01	maker	CDS	814	844	.	-	0	ID=EcMO1g00000010-mRNA-1:cds;Parent=EcMO1g00000010-mRNA-1
Chr01	maker	CDS	509	528	.	-	2	ID=EcMO1g00000010-mRNA-1:cds;Parent=EcMO1g00000010-mRNA-1
Chr01	maker	CDS	160	414	.	-	0	ID=EcMO1g00000010-mRNA-1:cds;Parent=EcMO1g00000010-mRNA-1
Chr01	maker	gene	11196	12704	.	-	.	ID=EcMO1g00000020;Name=EcMO1g00000020;description=Putative uncharacterized protein

Cheers
Yunus

**Comment by SJShaw** [comment]
What you've shown so far seems fine. What is the error being reported? Usually it provides some clues.

If it's just `could not parse records from GFF3 file`, then it's something do with with the features themselves. I've seen another maker GFF that had some start and end coordinates are reversed, so you could check for those. Otherwise, you'll need to check all the features.

I can look at making a change to add some more information about the error, but most of the errors raised from the library we use for parsing GFF files aren't particularly informative to begin with.
